block_my_feedback.php

```diff
@@ -114,11 +114,11 @@ public function fetch_marking(stdClass $user): ?array {
                 continue;
             }
             // Skip none current course.
-            if (!self::is_course_current($course)) {
+            if (!$this->is_course_current($course)) {
```

Comment:
Why don't we make `is_course_current()` a static function and then keep this call as `self::is_course_current($course)`? So:

```
            if (!self::is_course_current($course)) {
```

```
    public static function is_course_current(stdClass $course): bool {
```